### PR TITLE
feat: Automatically restart threads with increasing delay

### PR DESCRIPTION
**Pull request by LeagueOfPoro**
This PR gets rid of the three consecutively failed logins limit and instead implements increasing delay based on the number of failed logins. The delay is reset after a successful login.

### CODE REVIEW

File: src/Restarter.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        elif failedLogins == 2:
+            delay = 30
+        elif failedLogins == 3:
+            delay = 150
+        elif failedLogins == 4:
+            delay = 300
+        elif failedLogins == 5:
+            delay = 600
+        elif failedLogins >= 6:
+            delay = 1800
```
Comment (DjordyKoert) [inline src/Restarter.py:31]: I recommend to use a simple formula instead of lots of if/elif statement

Comment (LeagueOfPoro) [reply src/Restarter.py:31]: Unfortunately, a similar (exponential) formula would not be easy.

**Review by auroraisluna — COMMENTED** [review]
lgtm

File: src/Restarter.py (hunk 1 above)
Comment (chavan0) [reply src/Restarter.py:31]: I agree, 30 mins is too much. I think 5 minutes is tolerable on 6th attempt.

**Pull request merged**